db/migrate/20170531232835_add_game_id_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddGameIdToUser < ActiveRecord::Migration[5.0]
+  def change
+    add_column :users, :game_id, :integer
```

Comment:
If you do this, then each user can only play 1 game at a time.

We want a user to be able to play as many games as they want.

If you are looking for a `user.games` method, you can make one like this:

```Ruby
def games
  Game.where("white_player_id = ? OR black_player_id = ?", self.id, self.id)
end
```